doc/guide/build.hpp

```diff
@@ -46,61 +66,74 @@ system and have headers present:
  - Boost (math_c99, program_options, serialization, unit_test_framework, heap,
           spirit) >= 1.49
 
+For Python bindings, the following packages are required:
+
+ - setuptools
```

Comment:
Not sure, if this is necessary to note here, that ``pytest-runner`` is also needed for the tests? I guess in most cases it's going to be installed via the setuptools?